www/firebase.js

```diff
@@ -204,6 +204,24 @@ exports.clearAllNotifications = function (success, error) {
   exec(success, error, "FirebasePlugin", "clearAllNotifications", []);
 };
 
+exports.clearNotification = function (id, tag, success, error) {
+    var args = [id];
+    // "tag" is an optional arg
+    if (tag) {
+        if (typeof tag === 'function') {
```

Comment:
I'm confused, can we not pass null or something here and leave success/error callbacks in the right order?

Reply:
Could do, but we'd always need to remember that on the app which is risky. This approach isn't pretty but it's how the plugin does it in other places so I figured I'd stick with it.